### ISSUE TITLE
Oracle compatible types should be used without schema qualification

### ISSUE THREAD

**Issue by codeforall** [IvorySQL/IvorySQL#30]
## Enhancement
Creating a table with an Oracle compatible data type like Varchar2 requires schema qualification that is not very intuitive and also conflicts with the IvorySQL documentation.
IvorySQL should allow using these types without schema qualification.

### Current behavior
```SQL
CREATE TABLE test_ora_typw(a varchar2);
ERROR:  type "varchar2" does not exist
LINE 1: CREATE TABLE test_ora_typw(a varchar2);
                                     ^
-- Schema qualifying VARCHAR2 with Orcale works 
CREATE TABLE test_ora_typw(a oracle.varchar2);  
CREATE TABLE  
postgres=# 
```  

###Proposed enhancement
```SQL
-- Without schema qualification should work
CREATE TABLE test_ora_typw(a varchar2);
```

**Comment by BingAo-Office** [comment]
#31 Resolved this issue

**Comment by codeforall** [comment]
Fixed for V1.1 by [ PR#31](https://github.com/IvorySQL/IvorySQL/pull/31)
